tests/db/test_bigchain_api.py

```diff
@@ -143,6 +157,26 @@ def test_read_transaction_invalid_block(self, b, user_vk, user_sk):
         assert response is None
 
     @pytest.mark.usefixtures('inputs')
+    def test_read_transaction_valid_block(self, b, user_vk, user_sk):
+        input_tx = b.get_owned_ids(user_vk).pop()
+        tx = b.create_transaction(user_vk, user_vk, input_tx, 'TRANSFER')
+        tx_signed = b.sign_transaction(tx, user_sk)
+        b.write_transaction(tx_signed)
```

Comment:
I don't think there is a need to write the transaction since you are manually creating and voting on the block